docs_src/threat-models/secret-store/threat_model.md

```diff
@@ -15,7 +15,6 @@ and not the EdgeX project as a whole.
 The EdgeX Framework is a API-based software framework that strives to be platform and architecture-independent. The threat model considers only the following two deployment scenarios:
 
 * A containerized implementation based on Docker.
-* A confined implementation based on Snaps.
```

Comment:
The text **above and below** needs to be modified, since the change results in only one deployment scenario.